pypoptools/pypoptesting/framework/pop_util.py

```diff
@@ -39,9 +39,10 @@ def endorse_block(node: Node, apm, height: int, address: str = None) -> str:
         node.submitpopvbk(vbk_block.toVbkEncodingHex())
     for vtb in payloads.vtbs:
         node.submitpopvtb(vtb.toVbkEncodingHex())
-    node.submitpopatv(payloads.atv.toVbkEncodingHex())
+    for atv in payloads.atvs:
+        node.submitpopatv(atv.toVbkEncodingHex())
 
-    return payloads.atv.getId()
+    return payloads.atvs[0].getId()
```

Comment:
what if we have many ATVs here?

Reply:
Actually, `endorseAltBlock` always return exactly one ATV. 
Just because we now use `PopData` structure with collection `atvs`, not single field `atv`, we have such an issue.